Src/AutoNSubstitute.sln

```diff
@@ -11,6 +11,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoNSubstituteUnitTest", "
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoNSubstitute", "AutoNSubstitute\AutoNSubstitute.csproj", "{189C9474-B406-4356-A367-FBC882E9BA4B}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoFixture.xUnit.net", "AutoFixture.xUnit.net\AutoFixture.xUnit.net.csproj", "{266B9F0E-28E5-47CA-9816-ACB6737E7E4D}"
```

Comment:
Perhaps I wrote too soon when I wrote that [I'd proceed with a more detailed review](https://github.com/AutoFixture/AutoFixture/pull/430#discussion_r36284251). I'll first wait for the change that removes the scenario tests.

Will you please add a notification here when it's done? GitHub doesn't send me a notification when pull request branches are updated, but I *do* get a notification for every new comment.